framework/validators/FileValidator.php

```diff
@@ -258,7 +259,12 @@ protected function validateValue($file)
      */
     public function getSizeLimit()
     {
+        // Max POST size minus some bytes for form data 
+        $post_limit = $this->sizeToBytes(ini_get('post_max_size')) - 1024;
```

Comment:
since the purpose of the check is to give the user a feedback I don't think it's useful to just log a message. 
I think using the pure value is ok, it's really a corner case if the user is trying to upload a file which is *bytes* greater than the limit... Also, considering people usually refer to sizes in MB they won't even see this difference.